scipy/special/specfun.pyf

```diff
@@ -1,8 +1,6 @@
 !%f90 -*- f90 -*-
 python module specfun ! in
     interface  ! in :specfun
-        ! cpdsa
```

Comment:
I went ahead and deleted the comments too-I feel they mostly just make git grep return false positives